packages/wonder-blocks-tabs/src/components/tab-panel.tsx

```diff
@@ -13,20 +16,55 @@ type Props = {
      * The id of the associated element with role="tab".
      */
     "aria-labelledby": string;
+
+    /**
+     * Whether the tab panel is active.
+     */
+    active?: boolean;
 };
 
+const StyledDiv = addStyle("div");
 /**
  * A component that has `role="tabpanel"` and is used to represent a tab panel
  * in a tabbed interface.
  */
-export const TabPanel = React.forwardRef(function TabPanel(
-    props: Props,
-    ref: React.ForwardedRef<HTMLDivElement>,
```

Comment:
Removed support for forwarding a ref so we can use a ref internally to figure out if the tab panel has focusable elements in it. `TabPanel` is an internal component anyways used by `Tabs`, so consumers won't need to set a ref on the tab panel (if they needed to, they could put a ref on the contents of the panel, which they provide!)